src/store/model/store.ts

```diff
@@ -135,14 +145,18 @@ export type Model =
   | 'pulse'
   | 'red devil'
   | 'red dragon'
-  | 'sg oc'
+  | 'rog strix'
+  | 'rog strix oc'
+  | 'rog strix oc white'
```

Comment:
```suggestion
```

Do you mind using the `strix oc`, `strix`, etc. We removed the `rog` as it's part of the brand.

Reply:
the cleanup is done 👍
tested and ok.